packages/rum-recorder/src/domain/rrweb/record.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { createNewEvent } from '@datadog/browser-core'
+import { record } from './record'
+import { Event, EventType } from './types'
+
+describe('record', () => {
+  let input: HTMLInputElement
+  let stop: (() => void) | undefined
+
+  beforeEach(() => {
+    input = document.createElement('input')
+    document.body.appendChild(input)
+  })
+
+  afterEach(() => {
+    input.remove()
+    if (stop) {
+      stop()
+    }
+  })
+
+  it('will only have one full snapshot without checkout config', () => {
```

Comment:
As I said, I only included a single test, so we can discuss if we are comfortable with the current RRWeb integration before importing more tests.